### PR TITLE
fail2ban: include README files

### PR DESCRIPTION
**Pull request by iigs**
Include README files missing from initial package.

Fix comments in Makefile.

### CODE REVIEW

File: components/network/fail2ban/Makefile [hunk 1]
```diff
@@ -1,26 +1,33 @@
 #
-# This file and its contents are supplied under the terms of the
-# Common Development and Distribution License ("CDDL"), version 1.0.
-# You may only use this file in accordance with the terms of version
-# 1.0 of the CDDL.
+# CDDL HEADER START
```
Comment (mtelka) [inline components/network/fail2ban/Makefile]: Why this header change?  We prefer the previous one.  See https://github.com/OpenIndiana/oi-userland/tree/oi/hipster/templates

File: components/network/fail2ban/Makefile [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
-# A full copy of the text of the CDDL should have accompanied this
-# source.  A copy of the CDDL is also available via the Internet at
-# http://www.illumos.org/license/CDDL.
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
 #
-
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
```
Comment (mtelka) [inline components/network/fail2ban/Makefile]: ... because this URL is not reachable, for example.

File: components/network/fail2ban/Makefile [hunk 3]
```diff
@@ -63,7 +71,6 @@ COMPONENT_POST_INSTALL_ACTION += \
 	rm -f $(PROTO_DIR)/etc/fail2ban/paths-freebsd.conf ; \
 	rm -f $(PROTO_DIR)/etc/fail2ban/paths-opensuse.conf ; \
 	rm -f $(PROTO_DIR)/etc/fail2ban/paths-osx.conf ; \
```
Comment (mtelka) [inline components/network/fail2ban/Makefile]: You want to remove the ending backslash too.

**Comment by iigs** [comment]
What are the odds that I copy the CDDL header from a random Makefile in the repo and it is the older style one?  Copied header from a template .mk file. Removed backslash.  Changed the foo=bar lines to match a template .mk file.

**Review by mtelka — APPROVED** [review]

**Pull request merged**